EXILED/Exiled.API/Extensions/RoleExtensions.cs

```diff
@@ -123,6 +123,55 @@ public static bool TryGetRoleBase<T>(this RoleTypeId roleType, out T roleBase)
         /// <returns>Returns whether <paramref name="roleType"/> is an <see cref="IFpcRole"/> or not.</returns>
         public static bool IsFpcRole(this RoleTypeId roleType) => roleType.GetRoleBase() is IFpcRole;
 
+        /// <summary>
+        ///  Checks if the role is an SCP role.
+        /// </summary>
+        /// <param name="roleType">The <see cref="RoleTypeId"/>.</param>
+        /// <returns>A boolean which is true when the role is an SCP role.</returns>
+        public static bool IsScp(this RoleTypeId roleType) => roleType.GetTeam() == Team.SCPs;
+
+        /// <summary>
+        /// Checks if the role is a human role.
+        /// </summary>
+        /// <param name="roleType">The <see cref="RoleTypeId"/>.</param>
+        /// <returns>A boolean which is true when the role is a human role.</returns>
+        public static bool IsHuman(this RoleTypeId roleType) => !roleType.IsDead() && !roleType.IsScp();
+
+        /// <summary>
+        /// Checks if the role is a dead role.
+        /// </summary>
+        /// <param name="roleType">The <see cref="RoleTypeId"/>.</param>
+        /// <returns>A boolean which is true when the role is a dead role.</returns>
+        public static bool IsDead(this RoleTypeId roleType) => roleType.GetTeam() == Team.Dead;
+
+        /// <summary>
+        /// Checks if the role is an NTF role.
+        /// </summary>
+        /// <param name="roleType">The <see cref="RoleTypeId"/>.</param>
+        /// <returns>A boolean which is true when the role is an NTF role.</returns>
+        public static bool IsNtf(this RoleTypeId roleType) => roleType.GetTeam() == Team.FoundationForces;
```

Comment:
aren't guards also part of `Team.FoundationForces`?

Reply:
I think so, but guards are foundation forces anyway

Reply:
but they aren't Nine Tailed Fox, please correct

Reply:
Ok, I will remove them from Ntf